packages/server/utils/OpenAIServerManager.ts

```diff
@@ -17,6 +17,39 @@ class OpenAIServerManager {
     this.openAIApi = new OpenAIApi(configuration)
   }
 
+  async getStandupSummary(htmlResponses: string[], meetingPrompt: string) {
+    if (!this.openAIApi) return null
+    // :TODO: (jmtaber129): Include info about who made each response in the prompt, so that the LLM
+    // can include that in the response, e.g. "James is working on AI Summaries" vs. "Someone is
+    // working on AI Summaries".
+    const prompt = `Below is a list of responses submitted by team members to the question "${meetingPrompt}". If there are multiple responses, the responses are delimited by the string "NEW_RESPONSE". Identify up to 5 themes found within the responses. For each theme, provide a 2 to 3 sentence summary. In the summaries, only include information specified in the responses. When referring to people in the output, do not assume their gender and default to using the pronouns "they" and "them".
+
+    Desired format:
+    * <theme title>: <theme summary>
```

Comment:
+1 I like this format, but it'd be great if the title on the client is bold rather than having an asterisk. 

When there are lots of speech marks, it can be difficult to read. 

![Screenshot 2023-07-28 at 11 00 23](https://github.com/ParabolInc/parabol/assets/39854876/d3792419-4899-4ad1-ab67-f71eb2edeea9)

Reply:
Oooo this is a really good idea! However, this would require some additional complexity when rendering the summary (we could just ask chatGPT for an HTML output and then do `dangerouslySetInnerHTML`, but that could open us to XSS).

I've created https://github.com/ParabolInc/parabol/issues/8604 to track this enhancement

Reply:
Might be a stretch, but could we use markdown so `* ` gets converted to a bullet point? I'm finding it difficult to read with all the asterisks and quotes. If not, maybe a dash would be easier to interpret than an asterisk?

Reply:
@acressall absolutely! That's what https://github.com/ParabolInc/parabol/issues/8604 is intended to track - it's not a simple fix so I'd prefer to keep it out of this PR, but it would be great to prompt openAI for markdown or similar.

I went ahead and switched the asterisks to dashes per your recommendation, since it's definitely a bit cleaner and easier to read.